chef/cookbooks/updater/recipes/default.rb

```diff
@@ -23,6 +23,11 @@
   node.save
 
   if node[:platform_family] == "suse"
+
+    repos = Provisioner::Repositories.get_repos(node[:platform],
```

Comment:
not sure I understand the question. What's a 1st phase ? Anyway, I see the prepare-upgrade-scripts.rb script uses the same function call, so I think it should be safe for a mere update.